NorthstarDedicatedTest/ExploitFixes.cpp

```diff
@@ -5,11 +5,13 @@
 #include "NSMem.h"
 #include "cvar.h"
 
-typedef char(__fastcall* function_containing_emit_t)(uint64_t a1, uint64_t a2);
-function_containing_emit_t function_containing_emit;
-ConVar* sv_cheats;
+#define ENTITYLIST_SLOT_COUNT 4096
```

Comment:
All of the (not broken) native code seems to use 16384 (`0x4000`) when checking bounds for the entity array. Where does 4096 come from?

Reply:
https://developer.valvesoftware.com/wiki/Entity_index